core/src/main/java/com/linecorp/armeria/common/metric/MeterIdPrefixFunction.java

```diff
@@ -92,24 +92,22 @@ public MeterIdPrefix completeRequestPrefix(MeterRegistry registry, RequestLog lo
             private void buildTags(ImmutableList.Builder<Tag> tagListBuilder, RequestOnlyLog log) {
                 final RequestContext ctx = log.context();
 
-                String methodName = log.name();
-                if (methodName == null) {
-                    final RequestHeaders requestHeaders = log.requestHeaders();
-                    methodName = requestHeaders.method().name();
-                }
-
                 if (ctx instanceof ServiceRequestContext) {
                     final ServiceRequestContext sCtx = (ServiceRequestContext) ctx;
                     tagListBuilder.add(Tag.of(Flags.useLegacyMeterNames() ? "hostnamePattern"
                                                                           : "hostname.pattern",
                                               sCtx.config().virtualHost().hostnamePattern()));
                 }
 
+                String methodName = log.name();
+                if (methodName == null) {
+                    final RequestHeaders requestHeaders = log.requestHeaders();
+                    methodName = requestHeaders.method().name();
+                }
                 tagListBuilder.add(Tag.of("method", methodName));
-
-                if (ctx instanceof ServiceRequestContext) {
-                    final ServiceRequestContext sCtx = (ServiceRequestContext) ctx;
-                    tagListBuilder.add(Tag.of("route", sCtx.config().route().meterTag()));
```

Comment:
In retrospect, I thought `service` could replace the `route` tag. However, a service can be bound to multiple paths. Do I need to revive the `route` tag?

Reply:
In that case, a user could specify an alternative `defaultServiceName`?